src/editors/playgroundController.ts

```diff
@@ -539,56 +575,86 @@ export default class PlaygroundController {
     return this._transpile();
   }
 
-  async _transpile(): Promise<boolean> {
+  async getTranspiledContent(): Promise<{ namespace: ExportToLanguageNamespace, expression: string } | undefined> {
     const {
       textFromEditor,
       selectedText,
       selection,
-      importStatements,
       driverSyntax,
       builders,
       language
     } = this._exportToLanguageCodeLensProvider._exportToLanguageAddons;
-
-    log.info(`Start export to ${language} language`);
+    let namespace: ExportToLanguageNamespace = {
+      databaseName: 'DATABASE_NAME',
+      collectionName: 'COLLECTION_NAME'
+    };
+    let expression = '';
 
     if (!textFromEditor || !selection) {
-      void vscode.window.showInformationMessage(
-        'Please select one or more lines in the playground.'
-      );
-
-      return true;
+      return;
     }
 
-    try {
-      let transpiledExpression = '';
-      let imports = '';
-      let namespace: ExportToLanguageNamespace = {
-        databaseName: null,
-        collectionName: null
-      };
+    if (driverSyntax) {
+      const connectionId = this._connectionController.getActiveConnectionId();
+      let driverUrl = 'mongodb://localhost:27017';
```

Comment:
Why didn’t we need this before? I’m sure there is a reason, it’s just not obvious to me :)